### PR TITLE
[DPE-3160] Add awesome alert rules

### PR DESCRIPTION
**Pull request by Zvirovyi**

### CODE REVIEW

**Review by marceloneppel — COMMENTED** [review]
Nice work, Vladyslav! Could you describe the changes to the PR and the Jira ticket (I think https://warthogs.atlassian.net/browse/DPE-3160 is the one for those changes)?

File: src/prometheus_alert_rules/patroni_rules.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# This file based on https://samber.github.io/awesome-prometheus-alerts/rules#patroni-1
```
[the remark this answers is not in the document]
Comment (taurus-forever) [reply src/prometheus_alert_rules/patroni_rules.yaml:1]: @7annaba3l please share your ACK here.
Me: OK to keep it AS IS due "CC BY 4.0" [here](https://github.com/samber/awesome-prometheus-alerts/blob/master/LICENSE).

**Review by taurus-forever — APPROVED** [review]
Well done. Thank you! LGTM!
Please describe the PR chances to generate nice commit message. I am happy to merge this with the right PR message immediately!

**Comment by Zvirovyi** [comment]
Hi! Yes, this is [DPE-3160](https://warthogs.atlassian.net/browse/DPE-3160). I think best commit message would be just like the title or "add prometheus awesome alert rules". Thanks!

**Comment by taurus-forever** [comment]
Hi @Zvirovyi , the same story as [here](https://github.com/canonical/postgresql-k8s-operator/pull/357#issuecomment-1903518951). CI is not happy.

**Comment by Zvirovyi** [comment]
Hi, @taurus-forever. I think I fixed CLA

**Review by dragomirp — APPROVED** [review]

**Review by marceloneppel — APPROVED** [review]

**Comment by taurus-forever** [comment]
Yes. We can merge it!

**Pull request merged**